rebar.config

```diff
@@ -1,77 +1,81 @@
 %% == Erlang Compiler ==
 
-{minimum_otp_vsn, "19.3"}.
+{minimum_otp_vsn, "21.3"}.
+
 {erl_opts, [
-    warnings_as_errors,
     debug_info,
     warn_export_vars,
     warn_unused_import,
+    warnings_as_errors,
+    {i, "src"},
     {parse_transform, stacktrace_transform}
```

Comment:
Should we use this opportunity to get rid of `stacktrace_transform`? :)

Reply:
yes please

Reply:
We can, it wasn't in my plans to make it any more than OTP 24 compatible.

Reply:
d7c87a2

Reply:
[CI failed](https://github.com/miniclip/lhttpc/pull/24/checks?check_run_id=2023862027).

Reply:
And why not? 😄 I always get this, when I try to steer away from a pull request's initial scope. It's OK to take `stacktrace_compat` out, but if it wasn't done and CI was passing, it surely wasn't being harmful and could be done at a later stage.

Reply:
> could be done at a later stage.

As long as we have a process for that, sure. Otherwise it will be forgotten.